src/native/libs/System.Globalization.Native/pal_icushim.c

```diff
@@ -417,7 +419,7 @@ static void InitializeUColClonePointers(char* symbolVersion)
     ucol_safeClone_ptr = (ucol_safeClone_func)GetProcAddress((HMODULE)libicui18n, symbolName);
 #else
     char symbolName[SYMBOL_NAME_SIZE];
-    sprintf(symbolName, "ucol_safeClone%s", symbolVersion);
+    snprintf(symbolName, SYMBOL_NAME_SIZE, "ucol_safeClone%s", symbolVersion);
```

Comment:
>SYMBOL_NAME_SIZE [](http://example.com/codeflow?start=25&length=16)

I think this should be `MaxICUVersionStringLength + 1`

Reply:
The array is defined above as `SYMBOL_NAME_SIZE` characters, so `SYMBOL_NAME_SIZE` is the right value here.

Reply:
right. I overlooked that. please ignore.